packages/hoprd/src/api/v2/paths/account/alias.ts

```diff
@@ -0,0 +1,210 @@
+import type { Operation } from 'express-openapi'
+import type { State, StateOps } from '../../../../types'
+import PeerId from 'peer-id'
+import { STATUS_CODES } from '../../'
+
+/**
+ * Sets an alias and assigns the PeerId to it.
+ * Updates HOPRd's state.
+ * @returns new state
+ */
+export const setAlias = (stateOps: StateOps, alias: string, peerId: string): State => {
+  try {
+    const state = stateOps.getState()
+    state.aliases.set(alias, PeerId.createFromB58String(peerId))
+    stateOps.setState(state)
+    return state
+  } catch {
+    throw Error(STATUS_CODES.INVALID_PEERID)
+  }
+}
+
+/**
+ * Removes alias and it's assigned PeerId.
+ * Updates HOPRd's state.
+ * @returns new state
+ */
+export const removeAlias = (stateOps: StateOps, alias: string): State => {
+  const state = stateOps.getState()
+  state.aliases.delete(alias)
+  stateOps.setState(state)
+  return state
+}
+
+/**
+ * @returns The PeerId associated with the alias.
+ */
+export const getAlias = (state: Readonly<State>, alias: string): string => {
+  const peerId = state.aliases.get(alias)
+  if (!peerId) throw Error(STATUS_CODES.PEERID_NOT_FOUND)
+  return peerId.toB58String()
+}
+
+// @TODO: add get all aliases
+/**
+ * @returns All PeerIds keyed by their aliases.
+ */
+// export const getAliases = (state: Readonly<State>): { [alias: string]: string[] } => {
```

Comment:
let's try to get this in, maybe using the same route when parameter is not passed?